### ISSUE TITLE
ARM deployment of Service Fabric services gets stuck deploying when also defined in DefaultServices section in ApplicationManifest.xml

### ISSUE THREAD

**Issue by MedAnd** [Azure/service-fabric-issues#1581]
ARM deployment of Service Fabric services gets stuck deploying when the same service is also defined in DefaultServices section in ApplicationManifest.xml: 

![image](https://user-images.githubusercontent.com/13365135/65844701-9a8da480-e37a-11e9-91b3-cb4098a610d4.png)

Consider adding validation / feedback to ARM to cover this scenario. As many Service Fabric deployments will be using ApplicationManifest.xml originally, this finding would also be good clarify / add to the documentation?

> Any service being deployed via ARM json definition must be removed from the DefaultServices section of the corresponding ApplicationManifest.xml file.

https://github.com/MicrosoftDocs/azure-docs/issues/39485

**Comment by masnider** [comment]
This is correct. We'll update the docs and also hopefully make the experience better in case someone is using default services. @TassaduqBasu while I try to track down Marius' github membership :)

@BigMorty fyi as this is one of the conflicts we're running into a lot now regarding the VS tooling experience, since it isn't setting people up for success given all the new features in Azure.
